kibble/api/languages.go

```diff
@@ -50,6 +51,17 @@ func loadAllLanguagesFromApi(cfg *models.Config, site *models.Site) error {
 	site.DefaultLanguage = formatPathLocale(languages.DefaultLanguage.Code)
 	site.Languages = languages.mapToModel()
 
+	log.Infof("Site Languages Loaded from API")
+	var langaugesList []string
+	for _, langs := range site.Languages {
+		if langs.Code != "" {
+			langaugesList = append(langaugesList, langs.Code)
+		}
+	}
+	log.Infof("Default Language: %s ", site.DefaultLanguage)
+	if langaugesList != nil {
```

Comment:
I think it would be better if we were able to see this logged if it is somehow `nil`.

You can remove the `if` and just `log.Infof("Supported Languages: %v", languagesList)`